specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/EntityQueries.json

```diff
@@ -0,0 +1,651 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EntityQueryList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/entityQueries/{entityQueryId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get an Expansion entity query.": {
+            "$ref": "./examples/entityQueries/GetExpansionEntityQueryById.json"
+          },
+          "Get an Activity entity query.": {
+            "$ref": "./examples/entityQueries/GetActivityEntityQueryById.json"
+          }
+        },
+        "tags": [
+          "EntityQueries"
+        ],
+        "description": "Gets an entity query.",
+        "operationId": "EntityQueries_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/EntityQueryId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EntityQuery"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
New operations require a new api-version.  See the "breaking change process" in the PR description

Reply:
Exactly, the 2021-03-01-preview is the new api version,
this api exists in the 2019-01-01-preview.
did I miss something?

Reply:
2021-03-01-preview has already been published (https://armmanifest.visualstudio.com/One/_git/Manifest?path=%2FMICROSOFT.SECURITYINSIGHTS%2FPROD%2FMANIFEST.JSON&version=GBmaster&line=816&lineEnd=816&lineStartColumn=13&lineEndColumn=33&lineStyle=plain&_a=contents).  You cannot modify APIs in a public API version.  If you disagree please go through the breaking change exception process outlined in the PR description

Reply:
Yes, but this has been published via Manifest, which is not a customer facing contract.
if we had published the swagger before "opening" this API, customers would have tried to use it and get block by Arm.
I'll got via the breaking change exception if needed, but there is no record in the swagger regarding this api in the 2021-03-01 version

Reply:
I agree with @ori-licht.
@pilor - Three different resources were already added with a version `2021-03-01-preview` without issues -
https://github.com/Azure/azure-rest-api-specs/pull/13779
https://github.com/Azure/azure-rest-api-specs/pull/13998
https://github.com/Azure/azure-rest-api-specs/pull/13563

What is the difference between them and the current PR?
The PRs mentioned above were not marked with "NewApiVersionRequired" and the BreakingChange validation did not fail there. So the validation in this PR seems to be somehow wrong. 
`2021-03-01-preview` is the new API version of EntityQueries.

Reply:
If you feel it isn't a breaking change please go through the process outlined in the PR description.  I am just the messenger here, not the person that reviews breaking changes